Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -317,16 +320,10 @@ public override bool ExportExecutionDetailsToALM(BusinessFlow bizFlow, ref strin
                                 if (tsTest != null)
                                 {
                                     //get activities in group
-                                    List<Activity> activities = (bizFlow.Activities.Where(x => x.ActivitiesGroupID == activGroup.Name)).Select(a => a).ToList();
-                                    //Changed this because tsTest.Name is null. string TestCaseName = PathHelper.CleanInValidPathChars(tsTest.Name);
-
-                                    if ((publishToALMConfig.VariableForTCRunName == null) || (publishToALMConfig.VariableForTCRunName == string.Empty))
-                                    {
-                                        String timeStamp = DateTime.Now.ToString("dd-MMM-yyyy HH:mm:ss");
-                                        publishToALMConfig.VariableForTCRunName = "GingerRun_" + timeStamp;
-                                    }
+                                    List<Activity> activities = (bizFlow.Activities.Where(x => x.ActivitiesGroupID == activGroup.Name)).Select(a => a).ToList();                                    
 
-                                    CrateTestRun(publishToALMConfig, activGroup, tsTest, runSuite.Id, runFields);
+                                    //Commented below create test run as Above create test suite function creates test runs by default.
+                                    //CrateTestRun(publishToALMConfig, activGroup, tsTest, runSuite.Id, runFields);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6362823)